lib/components/SearchInput/SearchInput.module.scss

```diff
@@ -1,65 +1,64 @@
 @import '../../common/constants';
 @import '../../common/mixins';
 
-$search-max-width:400px;
```

Comment:
removing this will cause an error on the line 14

Reply:
this doesn't break because it lives in the constants file